package.json

```
@@ -98,7 +98,7 @@
     "release-it-lerna-changelog": "3.1.0"
   },
   "engines": {
-    "node": "12.* || >= 14"
+    "node": ">= 14"
```

```suggestion
    "node": "14.* || >= 16"
```

We don't want to support node 15 :) only LTS

---

Could you also please update readme?

---

Updated 😄